src/common/session/session-entity.js

```diff
@@ -32,7 +32,8 @@ const model = {
   loggingMode: LOGGING_MODE.OFF,
   serverTimeDiff: null, // set by TimeKeeper; "undefined" value will not be stringified and stored but "null" will
   custom: {},
-  numOfResets: 0
+  numOfResets: 0,
+  consent: false
```

Comment:
this looks good, but with the session entity we will just need to make sure that `consent`'s value get passed to the next session object when the session "resets"